### PR TITLE
Feature/loop 1088 change filters

### PR DESCRIPTION
**Pull request by martinyde**
https://jira.itkdev.dk/browse/LOOP-1090
- Removed text format selection.

https://jira.itkdev.dk/browse/LOOP-1088
- Added configuration for text formats

### CODE REVIEW

File: config/sync/filter.settings.yml [hunk 1]
```diff
@@ -2,3 +2,4 @@ fallback_format: plain_text
 always_show_fallback_choice: false
 _core:
   default_config_hash: FiPjM3WdB__ruFA7B6TLwni_UcZbmek5G4b2dxQItxA
+langcode: da
```
Comment (rimi-itk) [inline config/sync/filter.settings.yml:5]: Do we want this?

File: web/profiles/custom/os2loop/modules/os2loop_question/src/Helper/Helper.php [hunk 2]
```diff
@@ -56,8 +57,19 @@ public function alterForm(array &$form, FormStateInterface $form_state, string $
...
+      $form["os2loop_question_content"]["widget"][0]["#format"] = 'os2loop_question_rich_text';
+    }
+    else {
+      $form['os2loop_question_content']['widget'][0]['#better_formats']['settings']['allowed_formats'] =
+        ['os2loop_question_plain_text' => 'os2loop_question_plain_text'];
+      $form["os2loop_question_content"]["widget"][0]["#format"] = 'os2loop_question_plain_text';
+    }
+
+    $form['os2loop_question_content']['widget']['#after_build'][] =
+      [$this, 'fieldAfterBuild'];
```
Comment (rimi-itk) [inline web/profiles/custom/os2loop/modules/os2loop_question/src/Helper/Helper.php:72]: Do we need something similar for comments (both comments and answers)?

**Review by rimi-itk — COMMENTED** [review]
A few questions asked.

File: web/profiles/custom/os2loop/modules/os2loop_question/src/Helper/Helper.php (hunk 2 above)
Comment (martinyde) [reply web/profiles/custom/os2loop/modules/os2loop_question/src/Helper/Helper.php:72]: Added.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**